server/src/test/java/io/unitycatalog/server/base/function/BaseFunctionCRUDTest.java

```diff
@@ -101,7 +141,29 @@ public void testFunctionCRUD() throws ApiException {
                   .anySatisfy(parameter -> assertThat(parameter.getName()).isEqualTo("param1"));
             });
 
+    // List functions with page token
+    System.out.println("Testing list functions with page token..");
+    functionInfos =
+        functionOperations.listFunctions(CATALOG_NAME, SCHEMA_NAME, Optional.of(FUNCTION_NAME));
```

Comment:
Same comment as for `listCatalogs` -- can you do the test without assuming that the page token is the function name?